CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Update provisioning client struct member name in `az_iot_provisioning_client_register_response` from `registration_result` to `registration_state`.
 - Changed `operation_status` in `az_iot_provisioning_client_register_response` from `az_span` to `az_iot_provisioning_client_operation_status` enum.
 - Removed `az_iot_provisioning_client_parse_operation_status()` from `az_iot_provisioning_client.h`.
+- Renamed `AZ_ERROR_DEPENDENCY_NOT_PROVIDED` to `AZ_ERROR_HTTP_ADAPTER_DEPENDENCY_NOT_PROVIDED`.
+- Add new `az_result` value `AZ_ERROR_PLATFORM_DEPENDENCY_NOT_PROVIDED` which is returned by the no-op platform layer implementation.
```

Comment:
@antkmsft , @ahsonkhan  shouln't we use the same time for all sentences ?  We have present and past in here. Is that OK?
Update vs Changed vs Removed vs Add...

Reply:
Hmm, you are right, good point. We used "Add" in the past, just see the notes for the previous release below. To me, "Added" sounds better.

Reply:
Though it is good to be consistent (and if anyone wants to make the change, pleas do), it isn't something critical and difficult to enforce since folks have different styles of writing prose, without being strict. It, frankly, isn't worth the trouble.